code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -92,7 +92,7 @@
 
 /datum/species/monkey/handle_mutations_and_radiation(mob/living/carbon/human/H)
 	. = ..()
-	if(H.radiation > RAD_MOB_MUTATE * 2 && prob(50))
+	if(H.radiation > RAD_MOB_MUTATE * 2 && prob(0.5))
 		H.gorillize()
```

Comment:
```suggestion
	if(source.radiation > RAD_MOB_MUTATE * 2 && DT_PROB(0.5, delta_time))
		source.gorillize()
```

Updates to use DT_PROB macro.

Reply:
You need to halve the probability for delta_time to be a smooth conversion here, since the delta_time for life() is 2

Reply:
Good point, updated.